eo-maven-plugin/src/test/java/org/eolang/maven/optimization/OptCachedTest.java

```diff
@@ -74,6 +74,44 @@ void returnsFromCacheIfXmlAlreadyInCache(@TempDir final Path tmp) throws IOExcep
         );
     }
 
+    @Disabled
+    @Test
+    void returnsFromCacheButTimesSaveAndExecuteDifferent(@TempDir final Path tmp)
+        throws IOException, InterruptedException {
+        final XML program = OptCachedTest.program(ZonedDateTime.now());
+        Thread.sleep(70_000);
+        OptCachedTest.save(tmp, program);
+        MatcherAssert.assertThat(
+            "We expected that the program will be returned from the cache",
```

Comment:
@Yanich96, this message doesn't provide useful information for a developer. Perhaps you could explain what you specifically expect in this test case and add more context?

Reply:
@volodya-lombrozo is the message normal?
 "We expected that the program will be returned from the cache, " +
                "even if compilation and saving times are different"

Reply:
@Yanich96 Yes, I like it.